lib/processors/bundlers/flexChangesBundler.js

```diff
@@ -109,6 +133,9 @@ export default function({resources, options: {pathPrefix, hasFlexBundleVersion}}
 		const result = [];
 		if (nNumberOfChanges > 0) {
 			changesContent = sortAndStringifyInFlexFormat(changesContent);
+			if (hasFlexBundleVersion && Object.keys(existingFlexBundle).length > 0) {
+				changesContent = mergeFlexChangeBundles(changesContent, existingFlexBundle);
```

Comment:
It seems unfortunate that `sortAndStringifyInFlexFormat` stringifies an object, which `mergeFlexChangeBundles` will then have to parse again. I suppose the files are typically not very large and therefore there's no need to optimize this?

Otherwise maybe the `mergeFlexChangeBundles` could be moved into `sortAndStringifyInFlexFormat`, or the stringify could be moved outside `sortAndStringifyInFlexFormat`

Reply:
Done moved to sortAndStringifyInFlexFormat